woocommerce/woocommerce-google-analytics-integration
# Make up the missed changelog for version 1.5.19 and reduce the amount of changelog entries in readme.txt

### Changes proposed in this Pull Request:

Since the earlier release PR #244 for 1.5.19 is not yet merged, this PR
- Make up the missed changelog for version 1.5.19 and also move an entry from 1.6.0 to 1.5.19.
   - The moving entry "Fix - undefined WC constant" should belong to 1.5.19. 
   - See: [PR #242](https://github.com/woocommerce/woocommerce-google-analytics-integration/pull/242) and [version 1.5.19](https://github.com/woocommerce/woocommerce-google-analytics-integration/releases/tag/1.5.19)
- Reduce the amount of changelog entries in readme.txt to keep the last three versions, and add a link to the changelog.txt file on GitHub.

### Detailed test instructions:

1. View the updated [changelog.txt](https://github.com/woocommerce/woocommerce-google-analytics-integration/blob/4acacbed686f966641ed56ff0303c7bca5d2f97a/changelog.txt#L6-L19).
2. View the updated [readme.txt](https://github.com/woocommerce/woocommerce-google-analytics-integration/blob/4acacbed686f966641ed56ff0303c7bca5d2f97a/readme.txt#L61-L77).
3. Check if the updated changelog entries are all good.

### Changelog entry
